src/plots/gl2d/scene2d.js

```diff
@@ -522,6 +522,21 @@ proto.draw = function() {
                             glplot.pixelRatio
                 ];
 
+                // this needs to happen before the next block that deletes traceCoord data
+                // also it's important to copy, otherwise data is lost by the time event data is read
+                this.graphDiv.emit('plotly_hover', {
+                    points: [{
+                        trace: nextSelection.trace,
+                        dataCoord: nextSelection.dataCoord.slice(),
+                        traceCoord: nextSelection.traceCoord.slice(),
+                        textLabel: nextSelection.textLabel,
+                        color: nextSelection.color,
+                        name: nextSelection.name,
+                        hoverinfo: nextSelection.hoverinfo,
+                        screenCoord: nextSelection.screenCoord.slice()
```

Comment:
Oh nice. That looks pretty useful. 

But I would prefer aiming for parity with cartesian over event data as listed [here](https://github.com/plotly/plotly.js/blob/708e0ffd9ee827fd2640334a072f86605dcb2f5d/src/plots/cartesian/graph_interact.js#L477-L496) for now.

Referencing https://github.com/plotly/plotly.js/issues/988